doc/vic_installation/ts_tls_connection_error.md

```diff
@@ -0,0 +1,26 @@
+# Connection to Docker Client Fails with a TLS Connection Error #
+After a successful installation of vSphere Integrated Containers, connecting a Docker client to the virtual container host fails with a TLS connection error.
+
+## Problem ##
+After you have set the `DOCKER_HOST` variable to point to your virtual container host, when you attempt a Docker operation in your Docker client, the connection fails with the error `An error occurred trying to connect: Get https://<vic_appliance_address>:2376/v1.22/info: tls: oversized record received with length 20527`.
+
+## Cause ##
+The Docker client uses TLS authentication by default. The client is attempting to authenticate the connection with the virtual container host. TLS authentication is not activated on the virtual container host, or the Docker client does not have the appropriate certificate and key.
```

Comment:
Removed the "by default" bit. Added "Specify the `-tls` and `-tlsverify` options when you connect to the virtual container host. "